packages/next-app/src/pages/index.tsx

```diff
@@ -6,10 +6,19 @@ import React from "react";
 
 import { AssetFragment, useAssetsQuery } from "../../../hasura/dist/graphql";
 import { HasuraVariables, toHasuraCondition } from "../../../hasura/src/lib/hasura";
+import { INFINITE_SCROLL_NUMBER } from "../../../shared/src/configs/app";
 
 const HomePage: NextPage = () => {
   const [assets, setAssets] = React.useState<AssetFragment[]>([]);
-  const [variables, setVariables] = React.useState<HasuraVariables>({ where: {}, orderBy: {}, offset: 0, limit: 30 });
+  const [variables, setVariables] = React.useState<HasuraVariables>({
+    where: {},
+    orderBy: {},
+    offset: 0,
+    limit: INFINITE_SCROLL_NUMBER,
+  });
+  const [offset, setOffset] = React.useState(0);
+  const [hasMore, setHasMore] = React.useState(false);
```

Comment:
imho

I think this kind of boolean toggle, we can replace with useBoolean from chakra ui